pkg/exf/EXF_OPTIONS.h

```diff
@@ -98,12 +98,14 @@ C   >>> EXF_SEAICE_FRACTION <<<
 C       If defined, seaice fraction can be read-in from files (areaMaskFile)
 C
 C   >>> ALLOW_CLIMSST_RELAXATION <<<
-C       Allow the relaxation to a monthly climatology of sea surface
-C       temperature, e.g. the Reynolds climatology.
-C
+C       Allow the relaxation of surface level temperature to SST (climatology),
+C       e.g. the Reynolds climatology.
 C   >>> ALLOW_CLIMSSS_RELAXATION <<<
-C       Allow the relaxation to a monthly climatology of sea surface
-C       salinity, e.g. the Levitus climatology.
+C       Allow the relaxation of surface level salinity to SSS (climatology),
+C       e.g. the Levitus climatology.
+C   >>> ALLOW_CLIMSTRESS_RELAXATION <<<
+C       Allow the relaxation of surface-level horizontal-velocity to some
+C       prescribed 2-D fields (Note about name: does not involve any "stress")
```

Comment:
not only does it no involve stress, nor does it involve any relaxation! It should really be called `ALLOW_CLIMNOTSTRESS_ANDNORELAXATION` (or `ALLOW_SURFVEL_INSERTION` or similar).

```suggestion
C   >>> ALLOW_CLIMSTRESS_RELAXATION <<<
C       Allow the insertion of prescribed 3D surface-level horizontal-velocity,
C       but only possible if ALLOW_BULK_OFFLINE is defined (see exf_getclim.F)
C       (Note about name: does not involve any "stress")
```

Reply:
Oops, I though it was for surface velocity relaxation (similar to `ALLOW_CLIMSST/SSS_RELAXATION`), but you are right, it's only used within `ALLOW_BULK_OFFLINE` which I think should be removed completely (but not in this PR).

Will change/update my description.